<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>camIR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tourelle</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="CameraGB">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>130</height>
           </size>
          </property>
          <property name="title">
           <string>Tourelle</string>
          </property>
          <widget class="QLabel" name="SerialDef">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>128</width>
             <height>36</height>
            </rect>
           </property>
           <property name="text">
            <string>Serial Port</string>
           </property>
          </widget>
          <widget class="QLabel" name="AddrDef">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>128</width>
             <height>37</height>
            </rect>
           </property>
           <property name="text">
            <string>Receiver Address</string>
           </property>
          </widget>
          <widget class="QComboBox" name="SerialList">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>/dev/ttyS0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>/dev/ttyUSB0</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="CameraValida">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>90</y>
             <width>111</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="AddrList">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>297</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="PresetGB">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>130</height>
           </size>
          </property>
          <property name="title">
           <string>Preset</string>
          </property>
          <widget class="QComboBox" name="PresetMenu">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Set Preset</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Go To Preset</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Clear Preset</string>
            </property>
           </item>
          </widget>
          <widget class="QSpinBox" name="PresetNumber">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
          <widget class="QPushButton" name="ValidPreset">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>90</y>
             <width>111</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>57</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Action</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Preset number</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>297</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Manual</string>
          </property>
          <widget class="QPushButton" name="btnLeft">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Left (J)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnUp">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>10</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Up (I)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnDown">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>90</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Down (K)</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnRight">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>50</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Right (L)</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>297</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="thermacam">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>90</height>
           </size>
          </property>
          <property name="title">
           <string>Camera</string>
          </property>
          <widget class="QLabel" name="SerialDef_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>91</width>
             <height>36</height>
            </rect>
           </property>
           <property name="text">
            <string>Serial Port</string>
           </property>
          </widget>
          <widget class="QComboBox" name="serialThermaList">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>/dev/ttyS0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>/dev/ttyUSB0</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="thermacamValida">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>60</y>
             <width>111</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Zoom</string>
          </property>
          <widget class="QDoubleSpinBox" name="zoomPower">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>20</y>
             <width>81</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>8.000000000000000</double>
           </property>
          </widget>
          <widget class="QPushButton" name="btnZoom">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>20</y>
             <width>83</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="focusbox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Focus</string>
          </property>
          <widget class="QPushButton" name="btnFocusInf">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>83</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Infinity</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFocusClose">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>20</y>
             <width>83</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Close-Up</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFocusZoom">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>20</y>
             <width>83</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QGroupBox" name="rangebox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>Temperature Range</string>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Low Temp(°C)</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>High Temp(°C)</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="lowTemp">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>20</y>
             <width>81</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <double>-40.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="highTemp">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>50</y>
             <width>81</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <double>-40.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
          <widget class="QPushButton" name="setRange">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>20</y>
             <width>83</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>Set Range</string>
           </property>
          </widget>
          <widget class="QPushButton" name="autoRange">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>80</y>
             <width>111</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>AUTO RANGE</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QGroupBox" name="imageBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>140</height>
           </size>
          </property>
          <property name="title">
           <string>Image</string>
          </property>
          <widget class="QComboBox" name="getOrSave">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Save Image</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Get Image</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Remove Image</string>
            </property>
           </item>
          </widget>
          <widget class="QLineEdit" name="imgName">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>20</y>
             <width>151</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Name.jpg</string>
           </property>
          </widget>
          <widget class="QPushButton" name="imgAction">
           <property name="geometry">
            <rect>
             <x>132</x>
             <y>50</y>
             <width>151</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="imgList">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>83</y>
             <width>291</width>
             <height>51</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="overwriteMode">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>slot1()</slot>
 </slots>
</ui>
